frontend/src/metabase/components/Modal/WindowModal.tsx

```diff
@@ -43,7 +43,7 @@ export class WindowModal extends Component<WindowModalProps> {
     this._modalElement.className = "ModalContainer";
 
     if (props.zIndex != null) {
-      this._modalElement.setAttribute("style", `z-index:${props.zIndex}`);
+      this._modalElement.style.zIndex = String(props.zIndex);
```

Comment:
The previous syntax doesn't work anymore because it requires style parsing which is forbidden. Only using CSSOM API works